rem.backing/src/rem.backing.cpp

```diff
@@ -0,0 +1,76 @@
+#include <rem.backing.hpp>
+
+namespace eosio {
+
+   backing::backing(name receiver, name code,  datastream<const char*> ds)
+   :contract(receiver, code, ds),
+    rewards_dist_tbl(get_self(), get_self().value)
+    {
+      rewards_dist = rewards_dist_tbl.exists() ? rewards_dist_tbl.get() : rewarddist{};
+    }
+
+   void backing::distrewards()
+   {
+      claim_rewards( get_self() );
+      asset contract_balance = get_balance(token_account, get_self(), core_symbol);
+      check_rewards_dist_sum();
+
+      for (const auto &guardian: rewards_dist.reward_distribution) {
+         asset quantity = { static_cast<int64_t>(contract_balance.amount * guardian.second), core_symbol };
+         if (quantity.amount > 0) {
+            delegatebw(get_self(), guardian.first, quantity, true);
```

Comment:
I guess, due to precision, the last transfer may fail because it will lack 0.0001 REM, no?

Reply:
Good point, I fixed it in https://github.com/Remmeauth/rem-backing-contract/pull/8.

Reply:
@roman-tik, good point, I already fixed it in https://github.com/Remmeauth/rem-backing-contract/pull/8.